re.sonny.Commit.yaml

```diff
@@ -21,19 +21,26 @@ cleanup:
 modules:
   - name: gspell
     config-opts:
-      - '--disable-vala'
-      - '--disable-static'
-      - '--disable-gtk-doc'
+      - --disable-vala
+      - --disable-static
+      - --disable-gtk-doc
     sources:
       - type: archive
-        url: https://download-fallback.gnome.org/sources/gspell/1.9/gspell-1.9.1.tar.xz
-        sha256: dcbb769dfdde8e3c0a8ed3102ce7e661abbf7ddf85df08b29915e92cd723abdd
+        url: https://download.gnome.org/sources/gspell/1.8/gspell-1.8.4.tar.xz
```

Comment:
Why downgrading gspell?

Reply:
Where do you get that information from?

Reply:
I see - gspell README.md says it's following GNOME release logic

However, many distros ship gspell 1.9.1 https://repology.org/project/gspell/versions and that's what I'm testing/developping against.

Please revert

Reply:
Alright - I guess not.

A PR description explaining the logic behind the changes - would have been super helpful :+1:

Reply:
You can always edit the manifest if needed. It is also optional to accept merge requests from flathub bot.